test/seeds/offchainSeeds/offchain.match2.seed.ts

```diff
@@ -39,6 +39,8 @@ export const seededDemandAAlreadyAllocated = '807d1184-9670-4fb0-bb33-28582e5467
 export const seededMatch2WithAllocatedDemands = '27965a5f-f3dd-4110-82e7-68f59bb02c2e'
 export const seededMatch2AcceptedA = '347411d3-3750-49dd-a548-88e9f2616d9c'
 export const seededMatch2AcceptedFinal = '85a50fd9-f20f-4a61-a7e4-3ad49b7c3f21'
+export const seededMatch2CancellationId = '85a50fd9-f20f-4a61-a7e4-3ad49b7c3f22'
+export const seededMatch2CancellationId2 = '85a50fd9-f20f-4a61-a7e4-3ad49b7c3f23'
```

Comment:
```ts
const seededCancellations = ['uuid-id1', 'uuid-id2']
```

Reply:
Only concern - does this potentially make the code less readable? - and another comment below